src/OrchardCore/OrchardCore.Deployment.Abstractions/DeploymentTarget.cs

```diff
@@ -12,6 +15,8 @@ public DeploymentTarget(LocalizedString name, LocalizedString description, Route
             Route = route;
         }
 
+        public IEnumerable<SelectListItem> Formats { get; set; }
```

Comment:
How about having common Dictonary/Json like structure here to hold any kind of properties?

e.g. In future if we decide to add blob target, it could save connection string property